welcome.py

```
@@ -0,0 +1,337 @@
+            """
+        )
+
+        st.markdown(hero_css, unsafe_allow_html=True)
+
+        left_col_html = textwrap.dedent(
+            """
+            <div class="intro-lifecycle-sidecar" role="complementary" aria-label="Lifecycle guidance">
+                <div class="intro-lifecycle-sidecar__eyebrow">What you'll do</div>
+                <h5 class="intro-lifecycle-sidecar__title">Build and use an AI spam detector</h5>
+                <p class="intro-lifecycle-sidecar__body">
+                    In this interactive journey, you’ll build and use your own AI system, an email spam detector. You will experience the key steps of a development lifecycle, step by step: no technical skills are needed.
```

Ruff (0.14.0)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)